test/docs/testplans/vnet_to_vnet.md

```diff
@@ -0,0 +1,60 @@
+
+In these tests we want to leverage the snappi API to test some of the functional aspects of DUT. 
+By verifying and validating the scaling aspects of the switches (hardware/BMv2) 
+we are able to emperically baseline the scaling values. Additonally, we also 
+want to check some of the relevant counters to features like IPv4, IPv6, TCP, 
+UDP, common traffic stats etc.
+
+### Scope
+
+These test cases are targeted for testing the feature functionality of DASH implementation. The purpose of these test cases are to verify inbound/outbound/bidirectional traffic streams originating and terminating at the traffic generator. We plan to use the [OTG](https://github.com/open-traffic-generator) (Open traffic generator model) by leveraging the snappi APIs. 
+
+### Testbed
+
+The generic testbed setup may consist of two OTG connected to two ports of a switch (hardware-based or BMv2 software switch). 
+In this specifc testbed setup we will use [Ixia-c](https://github.com/open-traffic-generator/ixia-c) which is a modern, powerful, and API-driven open traffic generator model-based traffic tool that will generate traffic to observe the behavior of the device under test (DUT).
+
+### Testbed Topology 
+
+The tesbed is setup to emulate a VNET to VNET scenario and to test the mechanisms of VM to VM communication in VNET as shown below.
+
+![VM-VM-in-VNET](../../images/vm-to-vm-communication-vnet.svg)
+
+
+
+To emulate each VM in a VNET an Ixia-c port will be utilized. The BMv2 will be the DUT (This can later be replaced by a hardware-based appliance that connects to physical ports of a traffic generator)
+
+![ixia-c.drawio](../../images/ixia-c.drawio.svg)
+
+
+
+### Test Methodology
+
+Following test methodologies will be utilized to verify and validate the behavior of the switch
+
+- Traffic generator will be used to configure traffic streams that originate and/or terminate on traffic generator ports
+- Data traffic will be sent from VNET to VNET to verify performance properties like: CPS, flow, PPS, and rule scale.  These performance properties will be verified while  features like: VM to VM communication in VNET, route support, LPM support, ACL support are enabled.
+- Depending on the test cases, a combination of UDP traffic for IPv4, IPv6 addresses will be generated
+
+### Test Cases
+[TODO: Update Table]
+
+| Test Description                                             | Expected Result               |
+| :----------------------------------------------------------- | :------------------------ |
+| UDP IPv4 inbound  | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics  |
```

Comment:
Please specify what assumptions, if any, which you make about counters on the DUT. No counters exist to date. We should feed expectations into the working groups so we can track items. It is also OK to say "once counters are defined, the test-cases will be refined to incorporate them."